src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -76,4 +76,10 @@ Feature: Vodqa test
   @android @doubleTap
   Scenario: Validate that user is able to double tap on an element
     Given I login to vodqa application using valid credentials
-    Then I should be able to double tap on an element
\ No newline at end of file
+    Then I should be able to double tap on an element
+
+  #  CONFIG=./configs/vodqa_local_config.properties TAG=@scrollVertically PLATFORM=android ./gradlew run
+  @android @pinchAndZoom
+  Scenario: Validate that user is able to pinch and zoom on particular element
+    Given I login to vodqa application using valid credentials
+    Then I should be able to pinch and zoom on an element
```

Comment:
As suggested in steps file you can add one more feature line for zoom out
Then I should be able to pinch and zoom on an element
And I should be able to pinch and zoom out an element

You can create something like this

Reply:
Implemented above